src/datacontainer.js

```diff
@@ -191,6 +199,7 @@ class DataContainer {
     let result = {
       valid: ajvValidate(content),
       errors: ajvValidate.errors,
+      errorsText: this.validator.errorsText(ajvValidate.errors, {separator: '; '}),
```

Comment:
Aha, this confused me above when you were calling errorsText on what looked like a normal ajv validator.

Reply:
Ajv's API is really weird..

Reply:
(but I did confirm that this works, as I was discovering that upgrades aren't supported)